
try {
    if (err) throw err;
} catch(err) {

}


